### PR TITLE
POM cleanup

### PR DESCRIPTION
**Pull request by lintool**
+ binds `appassembler:assemble` to package phase explicitly so it'll run by default.
+ unbinds shader from package phase so it _won't_ run by default - speeds up build (fat jar is only needed if we want to build the Python bridge)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Anserini
 Build using Maven:
 
 ```
-mvn clean package appassembler:assemble
+mvn clean package
 ```
 
 The `eval/` directory contains evaluation tools and scripts, including `trec_eval`. Before using `trec_eval`, unpack and compile it.
```
Comment (Peilin-Yang) [inline README.md]: Can we add more specific commands here?
`tar xvfz trec_eval.9.0.tar.gz && cd trec_eval.9.0 && make`

File: pom.xml [hunk 2]
```diff
@@ -45,40 +45,37 @@
...
+            <configuration>
+              <extraJvmArguments>-Xms512M -Xmx24576M</extraJvmArguments>
+              <programs>
+                <program>
+                  <mainClass>io.anserini.index.IndexCollection</mainClass>
+                  <id>IndexCollection</id>
+                </program>
+                <program>
+                  <mainClass>io.anserini.search.SearchCollection</mainClass>
+                  <id>SearchCollection</id>
```
Comment (Peilin-Yang) [inline pom.xml]: I think you had accidentally removed `Eval` and `IndexUtils` in one recent PR?
Can you add them back?

**Comment by lintool** [comment]
Addressed CR.

File: pom.xml (hunk 2 above)
Comment (lintool) [reply pom.xml]: done

**Pull request merged**